src/_community/rules.md

```diff
@@ -46,14 +46,26 @@ Essentially, please take requests from community team members seriously and list
 something. If you have a problem with what a team member is asking you to do, we advise that you step away from the 
 situation and send a private message to another team member.
 
-#### 5) Keep all projects legal & appropriate
-
-If you need help with a project, you will not be provided with it if your project may break laws, breach a Terms of
-Service or EULA (belonging to any organisation, not just the platforms we're using), or be considered malicious or 
-inappropriate.
-
-Additionally, keep your code appropriate. Do not sexualize your code or use any slurs or other forms of derogatory
-language, either in the code itself or in the assets or filenames that make up your project.
+#### 5) Keep all projects legal, legitimate & appropriate
+
+While anyone may make use of the Quilt toolchain in accordance with the licenses applied to each project, all projects 
+developed or promoted in Quilt community spaces must meet the following criteria:
```

Comment:
I have got multiple people asking me whether this rule now means they have to follow these criteria when making Quilt mods in general. Is it possible to make "in Quilt community spaces" in bold or something to make it clearer?

Otherwise I'm worried about the effect it might have on people making mods for fun for private or research use (such as quite a few more responsible members of the seed cracking community). I have got multiple questions from them already worried about this rule before I had to clarify this point for them.

Reply:
I honestly feel like this is apparent already. I've still added a bold for you, but I really feel like people should actually be reading the thing. I'll push shortly.

Reply:
Not even 95% of the people will read rules, tos or privacy policy's. Because its to long/hard to read or they just don't care. 
Everything should be as short and clear as possible almost like short labels to see what you can may or may not do.

Reply:
Yes, I'm well aware of that. However, you'd think that people that are concerned about the rule would read the whole thing, no?

Reply:
Wouldn't that be lovely :laughing: but there are people who do at least scan rules, but then may misread them if it looks like they might say something else. I have this occasionally on my discord where I have a rule explicitly allowing politics.

Reply:
Fair enough, I guess, haha

Is the bold enough now?